projects/samples/robotbenchmark/maze_runner/worlds/maze_runner.wbt

```diff
@@ -51,6 +51,21 @@ Table {
   size 1.8 0.61 1
   feetSize 0.05 0.4
   feetPadding 0.1
+  trayAppearance PBRAppearance {
+    baseColorMap ImageTexture {
+      url [
+        "textures/table_top.jpg"
+      ]
+    }
+    roughness 1
+    metalness 0
+    textureTransform TextureTransform {
+      scale 1 1.7999999999999998
```

Comment:
```suggestion
      scale 1 1.8
```

Reply:
Thank you.
I set it to 1.8 from the GUI, but this has been automatically changed to 1.7999999999999998 when saving the file.

Reply:
Strange, I thought we fixed that problem some time ago...

Reply:
I would be nice to report it in a new (or old) issue.

Reply:
I will try to reproduce it and report it